examples/pytorch/IterativePruning/iterative_prune_alexnet.py

```diff
@@ -0,0 +1,191 @@
+# pylint: disable=W0221
+# pylint: disable=W0201
+# pylint: disable=W0223
+# pylint: disable=arguments-differ
+# pylint: disable=abstract-method
```

Comment:
Can you replace `pylint: disable=W0xxx`with `pylint: disable=<rule name>` syntax?